examples/api/lib/material/scaffold/scaffold_messenger_state.show_snack_bar.1.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for SnackBar
+
+import 'package:flutter/material.dart';
+
+void main() => runApp(const SnackBarApp());
+
+class SnackBarApp extends StatelessWidget {
+  const SnackBarApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(
+      home: SnackBarExample(),
+    );
+  }
+}
+
+class SnackBarExample extends StatelessWidget {
+  const SnackBarExample({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: const Text('SnackBar Sample')),
+      body: Container(
+        alignment: Alignment.topCenter,
+        padding: const EdgeInsets.all(16.0),
+        child: ElevatedButton(
+          onPressed: () {
+            const SnackBar snackBar = SnackBar(
+              content: Text('A SnackBar has been shown.'),
+              behavior: SnackBarBehavior.floating,
+            );
+            ScaffoldMessenger.of(context).showSnackBar(snackBar);
+          },
+          child: const Text('Show SnackBar'),
+        ),
+      ),
+      // Floating SnackBar are presented above Scaffold.floatingActionButton
+      // Avoid using widget with a large height in floatingActionButton
+      // as it might set a floating SnackBar off screen.
+      // Consider using constrained Widget's size in [Scaffold.floatingActionButton].
+      floatingActionButton: Container(
+        constraints: const BoxConstraints.tightFor(width: 150, height: 150),
```

Comment:
Is this example supposed to trigger the error? Maybe we can add more to this like, pressing the button changes the size of the floatingActionButton widget so folks can see the difference in where the SnackBar ends up.

Reply:
Great ! I added a second button to grow the floatingActionButton widget.